app/models/check.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Check < ApplicationRecord
```

Comment:
Perhaps we should add some documenting comments on this class to help future developer understand what this model is for and what it does.